Categories/Adjoint/Instance/0-Truncation.agda

```diff
@@ -0,0 +1,61 @@
+{-# OPTIONS --without-K --safe #-}
+
+module Categories.Adjoint.Instance.0-Truncation where
+
+-- The adjunction between 0-trucation and the inclusion functor from
```

Comment:
typo - truncation.

Reply:
Fixed — should spell-check my comments 😄